google-cloud-clients/google-cloud-contrib/google-cloud-spanner-jdbc/src/test/java/com/google/cloud/spanner/jdbc/it/ITSqlMusicScriptTest.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.spanner.jdbc.it;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.junit.runners.MethodSorters;
+import com.google.cloud.spanner.AbortedDueToConcurrentModificationException;
+import com.google.cloud.spanner.IntegrationTest;
+import com.google.cloud.spanner.Mutation;
+import com.google.cloud.spanner.ResultSet;
+import com.google.cloud.spanner.Statement;
+import com.google.cloud.spanner.jdbc.AbstractSqlScriptVerifier.GenericConnection;
+import com.google.cloud.spanner.jdbc.ITAbstractSpannerTest;
+import com.google.cloud.spanner.jdbc.SqlScriptVerifier;
+import com.google.cloud.spanner.jdbc.SqlScriptVerifier.SpannerGenericConnection;
+
+/**
+ * Integration test that runs one long sql script using the default Singers/Albums/Songs/Concerts
+ * data model
+ */
+@Category(IntegrationTest.class)
+@RunWith(JUnit4.class)
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class ITSqlMusicScriptTest extends ITAbstractSpannerTest {
+  private static final String SCRIPT_FILE = "ITSqlMusicScriptTest.sql";
+
+  @Test
+  public void test01_RunScript() throws Exception {
+    SqlScriptVerifier verifier = new SqlScriptVerifier();
+    try (GenericConnection connection = SpannerGenericConnection.of(createConnection())) {
+      verifier.verifyStatementsInFile(connection, SCRIPT_FILE, SqlScriptVerifier.class, true);
+    }
+  }
+
+  @Test
+  public void test02_RunAbortedTest() {
+    final long SINGER_ID = 2L;
+    final long VENUE_ID = 68L;
+    long numberOfSongs = 0L;
+    AbortInterceptor interceptor = new AbortInterceptor(0.0D);
+    try (ITConnection connection = createConnection(interceptor)) {
+      connection.setAutocommit(false);
+      connection.setRetryAbortsInternally(true);
+      // read all data from the different music tables in the transaction
+      try (ResultSet rs = connection.executeQuery(Statement.of("SELECT * FROM Singers"))) {
+        while (rs.next()) {
+          // do nothing
```

Comment:
let's add some basic assertions here on the result set? Can be simple like asserting the id, name, etc

Reply:
Added assertions for the ID's and number of records that are read from the database. The first test script adds but also deletes some of the records, so the assertions contain some if-statements checking for specific ID's that it knows have been deleted.